server/oracledb.js

```diff
@@ -0,0 +1,44 @@
+var settings = {};
```

Comment:
could refactor this and previous to be a oracle-base.js, that accepted the module name as an argument